src/goose/toolkit/developer.py

```diff
@@ -15,6 +16,10 @@
 from goose.toolkit.utils import get_language, render_template
 
 
+# Global dictionary to store the last read timestamps
```

Comment:
I don't think we need a global, this could be a member of the toolkit class? I think slightly better to have these separate per instance (if we were to ever run multiple goose threads)

Reply:
yes